test/js/node/dns/node-dns.test.js

```diff
@@ -190,3 +192,34 @@ test("dns.lookup (localhost)", done => {
     done(err);
   });
 });
+
+test("dns.getServers", done => {
+  function parseResolvConf() {
+    let servers = [];
+    try {
+      const content = fs.readFileSync("/etc/resolv.conf", "utf-8");
```

Comment:
I think this test will only work on linux 

can use test.skipIf(process.platform === “darwin”)